src/theme/ThemeProvider.js

```diff
@@ -1,6 +1,14 @@
 import React from 'react'
 import { ThemeProvider } from 'styled-components'
 
+const dropdown = {
+  background: '#06C1C0',
+  hover: '#D9FFF9',
+  neutral: '#BEBEBE',
+  scrollbar: '#615b82',
+  error: '#F18383',
+}
```

Comment:
Ideally these should re-use theme colours. 

maybe
`dropdown.background` -> `primary`
`dropdown.hover` -> make a new theme colour for this one I think
`dropdown.neutral` -> default
`dropdown.scrollbar` -> `foreground`
`dropdown.error` -> `secondaryWarning`

Reply:
@ianmah would using `linkHover` be fine for the hover state?

Reply:
I think so, just double check with design if you're unsure